server/rpc.go

```diff
@@ -203,11 +204,9 @@ func StartTranscodeServer(orch Orchestrator, bind string, mux *http.ServeMux, wo
 
 	glog.Info("Listening for RPC on ", bind)
 	srv := http.Server{
-		Addr:    bind,
-		Handler: &lp,
-		// XXX doesn't handle streaming RPC well; split remote transcoder RPC?
-		//ReadTimeout:  HTTPTimeout,
-		//WriteTimeout: HTTPTimeout,
+		Addr:        bind,
+		Handler:     &lp,
+		IdleTimeout: HTTPTimeout,
```

Comment:
Should we have the other timeouts too? Maybe even just the more general `Timeout`? I'm not sure there's any reason for a request to take longer than that

https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779

Reply:
In general we should, but currently it would break the split O/T functionality, because `/transcodeResults` endpoint would also have read/write timeouts and our transcoders do not tolerate it now...

So, in general you're right, but I think to make it happen it's not a quick fix.

Reply:
👍 Makes sense